card/lib/card/view/options/voo_api.rb

```diff
@@ -65,13 +57,13 @@ def slot_options
           end
         end
 
-        def closest_live_option key
-          if live_options.key? key
-            live_options[key]
-          elsif (ancestor = next_ancestor)
-            ancestor.closest_live_option key
-          end
-        end
+        # def inherit key
+        #   if live_options.key? key
+        #     live_options[key]
+        #   elsif (ancestor = next_ancestor)
+        #     ancestor.inherit key
+        #   end
+        # end
```

Comment:
I can still imagine a case where we might want this with legitimate options, but I refactored it away because it was only being used by foreign ones.